sdk/servicebus/service-bus/test/perf/azure-sb-package/send.ts

```diff
@@ -0,0 +1,125 @@
+/*
+# Overview
+Measures the maximum throughput of `sender.send()` in package `azure-sb`.
+
+# Instructions
+1. Create a Service Bus namespace with `Tier=Premium` and `Messaging Units=4`.  It is recommended to use the largest possible namespace to allow maximum client throughput.
+2. Create a queue inside the namespace.
+3. Set env vars `SERVICE_BUS_CONNECTION_STRING` and `SERVICE_BUS_QUEUE_NAME`.
+4. `ts-node send.ts [maxInflightMessages] [totalMessages]`
+5. Example: `ts-node send.ts 1000 1000000`
+ */
+
+import { createServiceBusService, ServiceBusService } from "azure-sb";
+import delay from "delay";
+import moment from "moment";
+
+const _payload = Buffer.alloc(1024);
+const _start = moment();
+
+let _sent = 0;
+let _accepted = 0;
+
+async function main(): Promise<void> {
+  // Endpoint=sb://<your-namespace>.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=<shared-access-key>
+  const connectionString = process.env.SERVICE_BUS_CONNECTION_STRING as string;
+  const entityPath = process.env.SERVICE_BUS_QUEUE_NAME as string;
+
+  const maxInflight = process.argv.length > 2 ? parseInt(process.argv[2]) : 1;
+  const messages = process.argv.length > 3 ? parseInt(process.argv[3]) : 10;
+  log(`Maximum inflight messages: ${maxInflight}`);
+  log(`Total messages: ${messages}`);
+
+  const writeResultsPromise = WriteResults(messages);
+
+  await RunTest(connectionString, entityPath, maxInflight, messages);
+
+  await writeResultsPromise;
+}
+
+async function RunTest(
```

Comment:
I don't see any `await` inside this function, so I think you should remove `async`.